modules/sqs/main.tf

```diff
@@ -65,10 +77,10 @@ resource "aws_sqs_queue" "internal_report_queue" {
 
 # Dead-letter queue
 resource "aws_sqs_queue" "internal_report_dlq" {
-  name                       = "${var.prefix}-orca-internal-report-deadletter-queue"
+  name                       = "${var.prefix}-orca-internal-report-deadletter-queue.fifo"
+  fifo_queue                 = true
```

Comment:
Per our discussion on FIFO and how a FIF) DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.